specification/devcenter/DevCenter/DevBox/models.tsp

```diff
@@ -0,0 +1,490 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-client-generator-core";
+import "../shared/models.tsp";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using Azure.ClientGenerator.Core;
+
+namespace DevCenterService;
+
+@doc("The operating system type.")
+@projectedName("csharp", "DevBoxOSType")
+@projectedName("java", "DevBoxOsType")
+@projectedName("python", "OSType")
+enum OsType {
+  @doc("The Windows operating system.") Windows,
+}
+
+@doc("Indicates whether hibernate is supported and enabled, disabled, or unsupported by the operating system. Unknown hibernate support is represented as null.")
+enum HibernateSupport {
+  @doc("Hibernate is enabled.") Enabled,
+  @doc("Hibernate is not enabled.") Disabled,
+  @doc("Hibernate is not supported by the operating system.") OsUnsupported,
+}
+
+@doc("Indicates whether owners of Dev Boxes in a pool are local administrators on the Dev Boxes.")
+@projectedName("csharp", "LocalAdministratorStatus")
+@projectedName("java", "LocalAdministratorStatus")
+@projectedName("python", "LocalAdministratorStatus")
+enum LocalAdminStatus {
```

Comment:
you are forgetting about JS, that is why i didn't used for all clients. Also we could not change the enum name because in our service we named it LocalAdminStatus, so the deserialization needs to know that

Reply:
oooh, be we don't have convinience api for js, so it doesn't matter.

Reply:
ok

Reply:
I've moved to client.tsp, but it seems like @@clientName is not yet supported by csharp emitter. Thread discussing about this: [Thang Le: Using @clientName decorator doesn't rename model](https://teams.microsoft.com/l/message/19:906c1efbbec54dc8949ac736633e6bdf@thread.skype/1708100498825?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1708100498825&teamName=Azure%20SDK&channelName=TypeSpec%20Discussion&createdTime=1708100498825)